### DISCUSSION TITLE
Errors may occur in commenting out calculation results.

### DISCUSSION THREAD

**Discussion by LM-1863** [community/community#110730] [Copilot Conversations]
### Select Topic Area

Bug

### Body

When I was writing the JavaScript code, GitHub Copilot predicted the result of the operation and showed the comment-out as a candidate, but the result was incorrect. Specifically, they are as follows.
`let a = 10, b = 1;
console.log( --a * ++b);`
The correct operating result for this code is "18". However, GitHub Copilot offered the following suggestion.
`let a = 10, b = 1;
console.log( --a * ++b); // 90`

**Comment by mrspecht** [comment]
GitHub Copilot is an amazing tool, but its autocomplete feature is far from being perfect. Copilot is kinda unpredictable, and may make silly mistakes like the one you described. The same goes for ChatGPT, which often suggests me code based on old examples (some 'var's here an there in JS)
